routers/router.ams1.yml

```
@@ -38,3 +38,14 @@
     remote_address: ams01.dn42.ventilaar.net
     remote_port: 30207
     public_key: 2VgphViJTNqQWlGII40Vfdjw+Uh6t/or1BHOQNDIS2A=
+
+- name: RAOULBROUNS-AMS01
+  asn: 4242421612
+  ipv6: fe80::5400:04ff:fecb:5ff1
+  multiprotocol: true
+  extended_nexthop: true
+  sessions: [ipv6]
+  wireguard:
+    remote_address: dn24bgp.raoulbrouns.nl
```

Hi @reulnl - did you perhaps transpose `24` and `42`?

```suggestion
    remote_address: dn42bgp.raoulbrouns.nl
```

```
;; ANSWER SECTION:
dn42bgp.raoulbrouns.nl.	300	IN	AAAA	2a05:f480:1400:b4:5400:4ff:fecb:5ff1
```

---

Apologies, you're right. I've just committed the correct version.

---

No problem, looks good to me now.